src/test/java/com/answerdigital/benhession/academy/answerkingweek2/services/CategoryServiceTest.java

```diff
@@ -0,0 +1,310 @@
+package com.answerdigital.benhession.academy.answerkingweek2.services;
+
+import com.answerdigital.benhession.academy.answerkingweek2.exceptions.ConflictException;
+import com.answerdigital.benhession.academy.answerkingweek2.exceptions.NotFoundException;
+import com.answerdigital.benhession.academy.answerkingweek2.model.Category;
+import com.answerdigital.benhession.academy.answerkingweek2.model.Item;
+import com.answerdigital.benhession.academy.answerkingweek2.model.OrderItem;
+import com.answerdigital.benhession.academy.answerkingweek2.repositories.CategoryRepository;
+import com.answerdigital.benhession.academy.answerkingweek2.request.AddCategoryRequest;
+import com.answerdigital.benhession.academy.answerkingweek2.request.UpdateCategoryRequest;
+
+import java.math.BigDecimal;
+import java.util.HashSet;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Our new checkstyle rules, don't allow .* imports. however, just tried running a check on this file and it comes back successful. So maybe does not apply to all.

Reply:
I think it might not work on Test files, will check it. @AudriusPutys